tests/manage/pv_services/test_expansion_snapshot_clone.py

```diff
@@ -143,6 +171,18 @@ def test_expansion_snapshot_clone(
             pvc_obj.resize_pvc(pvc_size_expand_1, True)
         log.info(f"Verified: Size of all PVCs are expanded to {pvc_size_expand_1}Gi")
 
+        # Verify thick provision by checking the image used size
+        if pvc_create_sc_type == "thick":
+            check_rbd_image_used_size(
+                pvc_objs=self.pvcs,
+                usage_to_compare=f"{pvc_size_expand_1}GiB",
+                rbd_pool=constants.DEFAULT_BLOCKPOOL,
+                expect_match=True,
+            )
+            log.info(
+                f"Verified thick provision after expanding the PVCs to {pvc_size_expand_1}GiB"
+            )
```

Comment:
how do we verifying or failing if its not thick provisioning?

Reply:
Fixed